virttest/test_setup.py

```diff
@@ -1767,3 +1769,69 @@ def _all_killed():
             wait.wait_for(_all_killed, timeout=60)
         except OSError:
             logging.warn("egd.pl is running")
+
+
+class StraceQemu(object):
+
+    """
+    Attach strace to qemu VM processes, if enable_strace is 'yes'.
+    It's useful to analyze qemu hang issue. But it will generate
+    a large size logfile if it is enabled, so compressed original
+    logfiles after strace process terminated to safe disk space.
+    """
+
+    def __init__(self, test, params, env):
+        self.env = env
+        self.test = test
+        self.params = params
+        self.tarball = "strace.tgz"
```

Comment:
Is the variable 'tarball' used as log tar, right ?
If so, It's better to rename it.